### ISSUE TITLE
Network Connection Telemetry

### ISSUE THREAD

**Issue by Brandon7CC** [redcanaryco/mac-monitor#28]
**At a high level -- can you summarize your request?**
* Network connection telemetry request from @jkennedyvz.
* Add domain level netconns and correlate to process telemetry

**What is the current alternative solution?**
* EDR
* Wireshark
* Little Snitch

**Are there "In-the-Wild" threats or corresponding ATT&CK techniques that exist for which this telemetry would be helpful?**
* Lots 😁

**Anything else?**
N/A

**Comment by jkennedyvz** [comment]
Background for this and #26 :
I'd like to demonstrate experiments for implant detection such as [this](https://www.elastic.co/security-labs/identifying-beaconing-malware-using-elastic) using pandas/scipy/jupyter, and without relying on software that is inaccessible to individuals.
